### PR TITLE
Fix resource page button focus

### PR DESCRIPTION
**Pull request by pirkkahuhtala**

### CODE REVIEW

File: azure/pull-request.pipeline.yml [hunk 1]
```diff
@@ -30,9 +30,9 @@ steps:
     CI=true yarn test
   displayName: 'Yarn run tests'
 
-# - script: |
-#    CI=true yarn test-cypress-with-new-test-api
-#   displayName: 'Yarn run cypress (e2e) tests'
+- script: |
```
Comment (minevala) [inline azure/pull-request.pipeline.yml:33]: These will probably fail at the moment. Should we leave them in comments for now?

File: src/components/language-select/LanguageSelect.scss [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+  position: relative;
+}
+
+.custom-language-select:focus-within {
+  box-shadow: none;
+}
+
+.custom-language-select:focus-within::after {
+  border: 2px solid var(--color-white);
+  bottom: -2px;
```
Comment (minevala) [inline src/components/language-select/LanguageSelect.scss:12]: I would add "display: block" just to avoid lineheight issues and just for clarity.

**Review by minevala — COMMENTED** [review]
Nice fix 👍 
I added a couple of suggestions.

File: src/components/language-select/LanguageSelect.scss (hunk 2 above)
Comment (pirkkahuhtala) [reply src/components/language-select/LanguageSelect.scss:12]: Ah yes. I'll fix it. Thanks. 👍

**Review by minevala — APPROVED** [review]
👍

**Pull request merged**